### PR TITLE
Stubs file extended

### PR DESCRIPTION
**Pull request by Bushikot**
Hi @gilek!
Your msgpack stubs package is most popular(and probably only one) on packagist.org, so I decided to extend it instead of creating new one. You might be surprised by all of this things added by me, because they are not mentioned in examples on readme page of official extension's repo. Still, you can easily find this classes in extension tests files, and also in this extended readme of some active PR: [link](https://github.com/msgpack/msgpack-php/blob/ad1c3b60fb7a996afae187e0861d7bfd869f8ccf/README.md).

In short - `MessagePackUnpacker` provides an ability to parse streams.

Hope you like that. Thanks in advance!

### CODE REVIEW

**Review by gilek — APPROVED** [review]
Hi @Bushikot,
thank you for your PR. I am surprised seeing that anyone is interested in my internal stubs. Of course I'm accepting your request.
